cfgov/unprocessed/css/brand-palette.less

```diff
@@ -24,6 +24,7 @@
 @gray:              #75787B;
 @gray-80:           #919395;
 @gray-50:           #BABBBD;
+@gray-40:           #b4b5b6; // Close to what was known as "Gray 50"
```

Comment:
This would require wide-ranging updates across the codebase to adjust `@gray-50` to `@gray-40`,  which I think would be best to deliver in its own PR. At that time this whole file can just be overwritten by what's in the cf generator.